src/views/HardwareInstallation.vue

```diff
@@ -380,9 +395,99 @@ export default {
 			showComponentRemoveModal: false,
 			modalComponent: undefined,
 			modalComponentName: "",
+			hasData: false,
+			selectManufacturer: undefined,
 		};
 	},
 	computed: {
+		isLoading() {
+			return this.$store.state.mqtt[
+				"openWB/system/configurable/devices_components"
+			];
+		},
+		getDeviceListGeneric() {
+			var generic_arr = [];
+			for (const element of Object.values(this.isLoading)) {
+				if (element.group.includes("generic")) {
+					generic_arr.push({
+						value: element.value,
+						text: element.text,
+					});
+				}
+			}
+			return generic_arr;
+		},
+		getDeviceListOther() {
```

Comment:
Die Umsetzung im Backend macht es hier unnötig kompliziert. Wie in https://github.com/openWB/core/pull/1683 vorgeschlagen lieber nur eine einheitliche Struktur verwenden. Im optimalen Fall getrennt nach Hersteller und Gerätetyp.

Reply:
ok. wird angepasst. Danke